documentation/release_notes.rst

```diff
@@ -11,14 +11,74 @@ creating efficient heterogeneous applications.
+  C++20 random access ranges.
+- Improved performance of ``oneapi::dpl::experimental::kt::gpu::inclusive_scan`` and added support for binary operator
+  and type combinations which do not have a SYCL known identity.
+- Improved performance of ``inclusive_scan_by_segment`` and ``exclusive_scan_by_segment`` with
+  device execution policies on GPUs.
+- Improved performance of set operation algorithms when using device policies: ``set_union``, ``set_difference``,
+  ``set_intersection``, ``set_symmetric_difference``.
+- Improved performance of search operations (e.g., ``find``, ``all_of``, ``equal``, ``search``, etc.), ``is_heap`` and
+  ``is_heap_until`` algorithms on Intel® Arc™ B-series GPU devices.
+- Removed requirement of GPU double support to use ``set_union``, ``set_difference``, ``set_intersection``, and
+  ``set_symmetric_difference`` on Windows operating systems.
+- Removed copy constructible requirements from value type for ``reduce`` and ``transform_reduce`` algorithms when used
+  with a host policy, and default constructible requirements from value type for ``reduce`` and ``transform_reduce``
+  when used with any execution policy.
+
+Fixed Issues
+------------
+- Fixed an issue with incorrect handling of different projections by range-based ``merge``.
+- Fixed ``equal`` returning a ``false`` for empty input sequences; now it returns ``true``.
+- Fixed a compilation error **SYCL kernel cannot use exceptions** occurring with libstdc++ version 10 when calling
+  range-based ``adjacent_find``, ``is_sorted`` and ``is_sorted_until`` algorithms using  device policies.
+- Fixed an issue with ``PSTL_USE_NONTEMPORAL_STORES`` macro having no effect.
+- Fixed a bug where ``unique`` when called with a device execution policy returned an incorrect result
+  iterator.
+- Fixed a bug in ``exclusive_scan``, ``inclusive_scan``, ``transform_exclusive_scan``, ``transform_inclusive_scan``,
+  ``exlusive_scan_by_segment``, and ``inclusive_scan_by_segment`` algorithms when using device policies with differing
+  ``InputIterator`` and ``OutputIterator`` types.
+- Fixed error in return value types for oneapi::dpl::ranges::minmax_element and oneapi::dpl::ranges::mismatch algorithms.
+- Fixed compile errors in ``set_union`` and ``set_symmetric_difference`` for hetero execution policies when the types of
+  the second data and results data are different.
+
 Known Issues and Limitations
 ----------------------------
 New in This Release
 ^^^^^^^^^^^^^^^^^^^
-- Calling ``histogram`` algorithm with a device execution policy may cause a segmentation fault in
-  Intel® oneAPI DPC++/C++ Compiler 2025.3 when compiling SYCL kernels for CPU devices.
-  To avoid this, define ``ONEDPL_DISABLE_HISTOGRAM_REGISTER_REDUCTION`` macro to a non-zero value
-  prior to including oneDPL header files.
+- The `set_intersection`, `set_difference`, `set_symmetric_difference`, and `set_union` algorithms with a device policy
```

Comment:
I think this was accidentally copied here from the new issues section in the last release. We have the following note above saying it is fixed, "Removed requirement of GPU double support to use ``set_union``, ``set_difference``, ``set_intersection``, and ``set_symmetric_difference`` on Windows operating systems.".

Reply:
Fixed.